### PR TITLE
Preparing for 1.0.0 release.

### PR DESCRIPTION
**Pull request by m4dc4p**
* Initial documenation
* Uploaded a test release.

Recommend viewing this as github will render it at https://github.com/tozny/e3db-cli/tree/jgbailey/docs

### CODE REVIEW

**Comment by m4dc4p** [comment]
Still need to add links to Ruby, Go and Java SDKs.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+## Registration
+
+Before you can use E3DB, you must register an account and receive API
+credentials. To register using the Tozny E3DB CLI, run:
+
+```
+$ e3db register <your email>
+```
+
+This will register an account using the email address given. By
```
Comment (ericmann) [inline README.md]: While this flag _is_ an optional feature, let's simplify our docs by not calling it out. We _want_ to make sure sharing is as easy as possible and providing guidance towards this additional flag might overcomplicate things. Let's take the opinionated stance that registration _should_ be public and leave the `--public=false` option for power users to discover elsewhere (i.e. when using the command line).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+in the record.
+
+When other parties share data with us, the `Writer` column will show the
+ID of the client that shared the data, rather than our own client ID.
+
+#### Filters
+
+Several filters are available for matching records. Each argument can take a comma-separated list of values:
+
+- **`-t`/`--type` - Retrieve records with the given content type.
```
Comment (ericmann) [inline README.md]: The double `**` seems to be interpreted as bold in the diff, and is rendered as a literal in the parsed Markdown. Perhaps leave just the `-` to get the bulleted list?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+In order to set up sharing, you must know the unique ID or email
+adddress of the client you wish to share with. Similarly, if others
+wish to share with you, they must know your unique ID or email
+address. (If you passed the `public=false` flag when registering, your
+email address cannot be used to share with you). To find the unique
+ID of your client, run `e3db info`.
+
+The E3DB client allows you to share records based on their content
+type. For example, to share all address book entries with another
+client (who registered as `beamish@gimble.com` and made their email
```
Comment (ericmann) [inline README.md]: Is this a real email address? Maybe use `info@tozny.com` here instead ... or Isaac's email (for public sharing of feedback).

File: README.md [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+our
+blog](https://tozny.com/blog/announcing-project-e3db-the-end-to-end-encrypted-database/).
+
+Tozny's E3DB provides a familiar JSON-based NoSQL-style API for reading,
+writing, and listing JSON data stored securely in the cloud.
+
+First a caveat: *Don’t store anything you want to keep (yet!).* This
+is a developer preview, and we might have to delete the database
+periodically. Please do try it out as an experiment, and over time, we
+will build it into a durable encrypted storage solution!
```
Comment (jamesjb) [inline README.md]: Should delete this paragraph IMO since we're gearing up for 1.0!

File: README.md (hunk 1 above)
Comment (jamesjb) [reply README.md]: Agreed, I like the example including it but keeping it opt-in.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+Record ID:           874b41ff-ac84-4961-a91d-9e0c114d0e92
+Record Type:         address_book
+Writer ID:           e04af806-3bbf-41d6-9ed1-a12bdab671ee
+User ID:             e04af806-3bbf-41d6-9ed1-a12bdab671ee
+
+Field                Value
+------------------------------------------------------------------------------
+phone                503-555-1212
+name                 John Doe
+```
```
Comment (jamesjb) [inline README.md]: The new CLI doesn't output in this format. Not sure if it's worth implementing or not.

File: README.md (hunk 3 above)
Comment (m4dc4p) [reply README.md]: Just a made up email (using words from "Jabberwocky" by Lewis Carrol).

**Review by jamesjb — APPROVED** [review]
LGTM!

**Pull request merged**